src/targets/axis_2400.ini

```diff
@@ -0,0 +1,49 @@
+# ********************************
+# Transmitter targets
+# ********************************
+[env:AXIS_THOR_2400_TX_via_UART]
+extends = env_common_esp32
+build_flags =
+	${env_common_esp32.build_flags}
+	${common_env_data.build_flags_tx}
+	-include target/AXIS_THOR_2400_TX.h
+	-D TARGET_AXIS_THOR_2400_TX=1
+	-D USER_SETUP_LOADED=1
+	-D CONFIG_DISABLE_HAL_LOCKS=1
+	-include $PROJECT_DIR/lib/SCREEN/TFT_eSPI_User_Setup.h
+	-D VTABLES_IN_FLASH=1
+	-O2
+upload_speed = 420000
+src_filter = ${env_common_esp32.src_filter} -<rx_*.cpp>
+lib_deps = 
+    ${env_common_esp32.lib_deps}
+    https://github.com/jonesfu/TFT_eSPI.git
```

Comment:
If this fork is specific to ExpressLRS I guess it should be under ExpressLRS github organization umbrella?

Reply:
It is fork from https://github.com/Bodmer/TFT_eSPI.git, the major change is use software SPI access because ELRS is high real-time service with ESP32 and cannot break by others.
If ExpressLRS fork it and we can use  like https://github.com/ExpressLRS/TFT_eSPI.git,  we are very appreciate!